projects/packages/videopress/src/client/admin/components/input/index.tsx

```diff
@@ -37,27 +31,65 @@ export const Input = ( {
 	);
 
 	const handleKeyUpEvent = useCallback(
-		( e: KeyboardEvent< HTMLInputElement > ) => {
+		( e: KeyboardEvent< HTMLInputElement | HTMLTextAreaElement > ) => {
```

Comment:
Something that I just noticed, `e.key` is `Enter` both on normal and numpad enter keys, but `e.code` can be `Enter` or `NumpadEnter`, so we need something like `if ( onEnter != null && [ 'Enter', 'NumpadEnter' ].includes( e.code ) ) {`

Reply:
We could jump to `e.key` if we want.

Reply:
Sure :+1:

Reply:
https://github.com/Automattic/jetpack/issues/26138